src/modules/NetworkGraph/data-components/Graph.tsx

```diff
@@ -97,62 +99,64 @@ const Graph: React.FC<GraphProps> = ({ className, search, ...props }) => {
               <div className="text-center" style={{ color: 'var(--grey-425)' }}>
                 <Text size="sm">
                   <em>{status === 'PROCESSING' && <p>Data is being gathered</p>}</em>
-                  {/* <em>{status === 'PENDING' && <Loading size="sm" />}</em> */}
+                  {/* <em>{status === 'PENDING' && <Loading size="sm" showMessage={false} />}</em> */}
                 </Text>
               </div>
             </Col>
           </Row>
         </Container>
-        <Container className="fr-mt-0">
-          <Row>
-            <Col>
-              <Breadcrumb>
-                <BreadcrumbItem href={`${process.env.NEXT_PUBLIC_BASE_PATH || ''}/`}>
-                  Twitter
-                </BreadcrumbItem>
-                <BreadcrumbItem href={`${process.env.NEXT_PUBLIC_BASE_PATH || ''}/`}>
-                  Explore narratives
-                </BreadcrumbItem>
-                <BreadcrumbItem
-                  href={`${process.env.NEXT_PUBLIC_BASE_PATH || ''}/searches/${encodeURIComponent(
-                    search
-                  )}`}
-                >
-                  {search}
-                </BreadcrumbItem>
-                <BreadcrumbItem>Network of interaction graph</BreadcrumbItem>
-              </Breadcrumb>
-            </Col>
-          </Row>
-        </Container>
-        {!loading && error ? (
-          <div className={classNames(s.noData, className)} {...props}>
-            <Alert type="error">{error.toString()}</Alert>
-          </div>
-        ) : loading ? (
-          <Loading />
-        ) : (
-          <>
-            <Container className="fr-mb-6w">
-              <Row>
-                <Col>
-                  <Alert size="small">
-                    Due to the amount of data processed, the graph generation{' '}
-                    <strong>can take several minutes</strong> (be patient) and this requires a{' '}
-                    <strong>recent machine to be used properly</strong> (on mobile it is not
-                    feasible).
-                  </Alert>
-                </Col>
-              </Row>
-              <pre>{JSON.stringify(json?.metadata, null, 2)}</pre>
-            </Container>
-            {[undefined, 'PENDING', 'PROCESSING'].includes(status) && <Loading />}
-            {status === 'DONE_ERROR' && <Alert type="error">{data?.searchGraph.error}</Alert>}
-            {status === 'DONE' && <GraphDetail colors={dsfrColors} name={search} json={json} />}
-          </>
-        )}
-        {loading && <Loading />}
       </div>
+      <Container className="fr-mt-0">
+        <Row>
+          <Col>
+            <Breadcrumb>
+              <BreadcrumbItem href={`${process.env.NEXT_PUBLIC_BASE_PATH || ''}/`}>
```

Comment:
This BreadCrumbItem should be included in a <Link>

```
<Link href="/">
<BreadcrumbItem />
</Link>
```

But I'm not sure Breadcrumb will work without href passed (maybe css is based on this value)

You can use `passHref` on Link but it will still refresh the page when clicked so I would advise against

Reply:
I think it's not possible with this `react-dsfr`components.

2 solutions :
- keep it like this, reloading the it doesn't seem very binding to me
- using our Breadcrumb component that allow to use `next/Link`

@martinratinaud WDYT ?

Reply:
I'd rather
- use ours
- create an issue in react-dsfr repo

We won't be the only ones to have this problem

We used to have ours, right?